packages/blobstorage/indexer/indexer.go

```diff
@@ -291,31 +284,23 @@ func (i *Indexer) checkReorg(ctx context.Context, event *taikol1.TaikoL1BlockPro
 }
 
 func (i *Indexer) storeBlob(ctx context.Context, event *taikol1.TaikoL1BlockProposed) error {
-	slog.Info("blockProposed event found", "blockID", event.Meta.L1Height+1, "emittedIn", event.Raw.BlockNumber, "blobUsed", event.Meta.BlobUsed)
-
-	if !event.Meta.BlobUsed {
-		return nil
-	}
-
-	blockID := event.Meta.L1Height + 1
-	url := fmt.Sprintf("%s/%v", i.beaconURL, blockID)
-	response, err := http.Get(url)
+	blockID, err := i.beaconClient.timeToSlot(event.Meta.Timestamp)
 	if err != nil {
 		return err
 	}
-	defer response.Body.Close()
 
-	body, err := io.ReadAll(response.Body)
-	if err != nil {
-		return err
+	slog.Info("blockProposed event found", "blockID", blockID, "emittedIn", event.Raw.BlockNumber, "blobUsed", event.Meta.BlobUsed)
+
+	if !event.Meta.BlobUsed {
+		return nil
 	}
 
-	var responseData Response
-	if err := json.Unmarshal(body, &responseData); err != nil {
+	BlobsResponse, err := i.beaconClient.getBlobs(ctx, blockID)
```

Comment:
can we lowercase this, "blobsResponse", for consistency with other variable names